### PR TITLE
finished magic ball task

### PR DESCRIPTION
**Pull request by lamaconda**

### CODE REVIEW

**Review by VasiliZas — APPROVED** [review]

**Review by KaydunovDenis — APPROVED** [review]

**Review by PavelBoPavel — APPROVED** [review]
+

**Review by trump-ny — APPROVED** [review]

**Review by Puzzletania — APPROVED** [review]

File: homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return magicBall;
+    }
+
+    public int randomNumberOfPrediction() {
+        return random.nextInt(20);
+    }
+
+    public void printPrediction() {
+        log.info("Prediction: {}", predictionsMap().get(randomNumberOfPrediction()));
+    }
```
Comment (max-java) [inline homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java:45]:
```
predictionsMap().get(randomNumberOfPrediction()));
```
could be encapsulated in a single method, i.e. predictNext()

**Review by max-java — APPROVED** [review]
run sonar verification on this. Minor comment, otherwise looks good

File: homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import java.util.Random;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MagicBall {
+    Logger log = LoggerFactory.getLogger(MagicBall.class);
+    private final Random random = new Random();
+
+
+    public Map<Integer, String> predictionsMap() {
```
[the remark this answers is not in the document]
Comment (lamaconda) [reply homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java:14]: understood!

File: homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java (hunk 1 above)
Comment (lamaconda) [reply homework/shelkovich_ilya/magic8Ball/src/main/java/com/tutrit/java/quickstart/service/MagicBall.java:45]: got it!

**Pull request merged**